### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by melaxon** [webworker01/komodophp#1]
When ran ```composer require webworker01/komodophp```  i received error message:
```
[UnexpectedValueException]                                                                                                           
  Error while installing webworker01/komodophp, composer-plugin packages should have a class defined in their extra key to be usable.  
```

**Comment by webworker01** [comment]
Please try now, sorry for the trouble installing.

**Comment by webworker01** [comment]
I'll assume this is resolved because of #2
